components/post_view/post_header/index.ts

```diff
@@ -24,14 +31,26 @@ function mapStateToProps(state: GlobalState, ownProps: Props) {
     }
 
     const user = getUser(state, ownProps.post.user_id);
+    const currentUser = getCurrentUser(state);
+    const customStatus = user ? getCustomStatus(state, user.id) : {};
```

Comment:
Should this user be `ownProps.post.user_id`?
```suggestion
    const customStatus = getCustomStatus(state, ownProps.post.user_id) || {};
```

Reply:
`ownProps.post.user_id` is same as `user.id`.